test/unit_tests/braket/default_simulator/test_simulator.py

```diff
@@ -64,3 +66,11 @@ def test_base_local_simulator_abstract():
     )
     with pytest.raises(TypeError, match=abstract_methods):
         BaseLocalSimulator()
+
+
+def test_discontinous_qubits_local_simulator():
```

Comment:
1. This test will logically fit better to [test/unit_tests/braket/default_simulator/test_state_vector_simulator.py](https://github.com/amazon-braket/amazon-braket-default-simulator-python/pull/262/files#diff-a001e5150c262c19f9a8d0a6f4ff36a89a69b7d54a271cc813eb92cc7c37c9b8), please move it there.
2. A similar test explicitly calling the density matrix simulator will be needed, please create one in [test/unit_tests/braket/default_simulator/test_density_matrix_simulator.py](https://github.com/amazon-braket/amazon-braket-default-simulator-python/pull/262/files#diff-5e09c70a37d6a97c8df5c931ad46b987e8612bf1dde51e17ac881cf4130a416e)